packages/web-components/src/components/notice-choice/notice-choice.ts

```diff
@@ -611,7 +632,7 @@ class NoticeChoice extends StableSelectorMixin(LitElement) {
           </div>
           <div part='${prefix}--nc__post-text' class="${prefix}--nc__post-text"
           >${this.postTextTemplate()}</div>
-          <input type='hidden' id="preventFormSubmission" name="preventFormSubmission" value=${
+          <input part="input" type='hidden' id="preventFormSubmission" name="preventFormSubmission" value=${
```

Comment:
Hidden inputs aren't rendered. My understanding is the motivation behind shadows parts are for visual adjustments. If we needed to get this element in Javascript, we don't need a `part` attribute for that. Lets leave the `part` attribute out for this hidden element.